Tests/GoogleMapsUtilsSwiftTests/unit/Geometry/Models/GMUPolygonTest1.swift

```diff
@@ -0,0 +1,56 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+
+@testable import GoogleMapsUtils
+
```

Comment:
Doesn't this require an import of GoogleMaps to access GMSMutablePath?

Reply:
And CoreLocatin to get CLLocation*?

Reply:
In my local tests are passing, using Xcode 15.4, do you want me to add those import, for a safer side?